test/correctness/simd_op_check.cpp

```diff
@@ -1642,24 +1643,6 @@ void check_hvx_all() {
     check("v*.h = vasr(v*.w,v*.w,r*):sat", hvx_width/2, i16c((i32(u16_1) + i32(u16_2)) >> 8));
```

Comment:
While you are deleting code, I think all but the vmpa are now "live" tests! These commented off vasr tests can be removed now.